R/utils.R

```diff
@@ -380,3 +380,84 @@ defunction <- function(x) {
 get_unique_labels <- function(labels) {
   make.unique(gsub("[^[:alnum:]]", "_", tolower(labels)), sep = "_")
 }
+
+#' Create `renv` `.lock` file
+#'
+#' @description
+#' Function allows to create `renv::snapshot()` during the `teal` initialization.
+#'
+#' @details
+#'
+#' The `.lockfile` creation is based on [renv::snapshot()]. It allows multiple ways of `.lockfile` creation.
```

Comment:
```suggestion
#' Create `renv` `.lock` file
#'
#' Function allows to create `renv::snapshot()` during the `teal` initialization.
#'
#' The `.lockfile` creation is based on [renv::snapshot()]. It allows multiple ways of `.lockfile` creation.
```

just to keep it consistent with other documentation.

Reply:
thanks!